features/financial_assistance/step_definitions/applicant_personal_info_page_steps.rb

```diff
@@ -37,6 +37,11 @@
   find(:xpath, "//div[@class='selectric-scroll']")
 end
 
+Then(/form should not submit due to required relationship error$/) do
+  expect(page).to have_content 'Select relationship type'
+end
+
 And(/user fills in the missing relationship$/) do
+  find("span", :text => "choose").click
```

Comment:
1. please use class selector or page object
2. don't hard code english into specs

Reply:
thank you mark, I was already working on it. Changes will be available in next commit.